src/ASBNApp.DataAPI/Program.cs

```diff
@@ -50,21 +65,34 @@
         modelBuilder.GetEdmModel()));
 
 builder.Services.AddDbContext<ASBNAppContext>(
-    options => options.UseSqlServer(config.GetConnectionString("Default")));
+    options => options.UseSqlServer(builder.Configuration.GetConnectionString("DatabaseConnection")));
 
 // Finalizing
 var app = builder.Build();
 
+// more forward headers stuff
+app.UseHttpLogging();
+
+app.Use(async (context, next) =>
+{
+	// Connection: RemoteIp
+	app.Logger.LogInformation("Request RemoteIp: {RemoteIpAddress}",
+		context.Connection.RemoteIpAddress);
+
+	await next(context);
+});
```

Comment:
This code should log always the IP of the reverse proxy. If this is still usefull to you keep it. If it was just for experimenting around maybe remove it

Reply:
Good catch, meant to remove that (have done now).